docs/higher-order/tca.ipynb

```diff
@@ -0,0 +1,1448 @@
+{
```

Comment:
<p>We need to add intuition as to how to select rank here. I propose to add the following after the plot:</p><p> </p><blockquote>TCA will identify correlation across neurons, trials and time to build a number of components that maximize the ability to reconstruct the original data matrix.</blockquote><blockquote>It is therefore essential to use scientific judgement when using this tool. There are two main decision to take: The <strong>final rank</strong> to use and making sure that the extracted model is <strong>faithfully representing</strong> the neuronal activity.</blockquote><blockquote><br></blockquote><blockquote>Choosing a final rank is a complex question with no "one-size fits all" answer. Generally, as the rank increases, the objective will decrease as more degrees of freedom allows for better reconstructions. Higher rank models also are more computational expensive to compute.</blockquote><blockquote><br></blockquote><blockquote>In some cases, if the rank is too low, you could end up with unreliable models.  The model similarity plot is useful to make this evaluation as  each dot represents a given model run. A good practice is to pick a rank high enough that provides reliable, similar models at each run. Beyond that, you should use your scientific judgement and choose a rank that allow you to conduct your analysis in practice. If your goal is to identify broader components shared across many neurons, a lower rank might be more appropriate. Increasing the rank will further sub-divide components, improving the objective, but will not provide additional insights. Eventually you can test the sensibility of your analysis against this choice with cross-validation.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/307/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>